.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,11 @@
+# SPDX-FileCopyrightText: 2025 Sébastien Helleu <flashcode@flashtux.org>
```

Comment:
Looks good to me. 👍🏻 

You could add yourself here as well. But there is no obligation.